src/test/java/software/amazon/awssdk/crt/test/HttpClientConnectionManagerTest.java

```diff
@@ -208,6 +209,7 @@ public void testParallelRequests() throws Exception {
 
     @Test
     public void testMaxParallelRequests() throws Exception {
+        Log.initLoggingToFile(LogLevel.Trace, "test.log");
```

Comment:
Was this temporary?

Reply:
No, it's accidental, will remove, git add -u GOT ME